runtime/js/41_prompt.js

```diff
@@ -42,7 +42,7 @@ function prompt(message = "Prompt", defaultValue = "") {
   }
 
   return ops.op_read_line_prompt(
-    `${message} `,
+    `${yellow(bold(`${message}`))} `,
```

Comment:
There's a bug on Windows where op_read_line_prompt doesn't properly calculate the text width due to the ansi codes and so the cursor will be way off to the left.

Reply:
Yeah, if anything we should probably do this in Rust, but need to verify if that doesn't cause problems either.

Reply:
![image](https://github.com/denoland/deno/assets/13602871/77cf5c4a-73dd-4ac8-b4e8-8a022d6e22e6)